back/src/Guide/Api/Migration.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Guide.Api.Migration where
+
+import Imports
+
+import Hasql.Session (Session)
+-- import Hasql.Statement (Statement(..))
+import Hasql.Connection (Connection, Settings)
+
+import qualified Hasql.Session as HS
+-- import qualified Hasql.Decoders as HD
+-- import qualified Hasql.Encoders as HE
+import qualified Data.ByteString as B
+import qualified Hasql.Connection as HC
+
+
+main :: IO ()
+main = do
+  conn <- connection
+  dbCreate conn
+
+connection :: IO Connection
+connection = do
+  eiConnection <- HC.acquire connectionSettings
+  let conn = either (error . show) id eiConnection
```

Comment:
I would suggest:

```haskell
connection =
  either (error . show) pure =<< 
  HC.acquire connectionSettings
```

Why? Because this way the `error` will be thrown even if the connection is not used, instead of being thrown lazily.